spec/src/main/asciidoc/portability.asciidoc

```diff
@@ -1,22 +1,46 @@
 
-=== Portability in Relational Databases
+=== Portability for Relational Databases
 
 All functionality defined by Jakarta Data must be supported when using relational databases.
 
-1. **Support for Jakarta Persistence Annotations:** Jakarta Data, when used in conjunction with a Jakarta Persistence provider, ensures comprehensive support for all Jakarta Persistence entity annotations. This support covers a wide range of functionality, including entity definitions, primary keys, and relationships.
+==== Jakarta Persistence Annotations
+
+Jakarta Data, when used in conjunction with a Jakarta Data provider that is backed by Jakarta Persistence, requires support for the `jakarta.persistence.Entity` annotation on entity classes and requires support for the following minimal set of Jakarta Persistence annotations for entity properties:
+
+* `jakarta.persistence.Basic`
+* `jakarta.persistence.Column`
+* `jakarta.persistence.Convert`
+* `jakarta.persistence.ElementCollection`
+* `jakarta.persistence.Embedded`
+* `jakarta.persistence.Enumerated`
+* `jakarta.persistence.GeneratedValue`
+* `jakarta.persistence.Id`
+* `jakarta.persistence.Temporal`
+* `jakarta.persistence.Version`
+
+Jakarta Data providers might choose to offer additional functionality that is provided by Jakarta Persistence beyond what is stated in this section.
+
+==== Built-In Repositories
+
+Jakarta Data's built-in repositories, such as `PageableRepository` and `BasicRepository`, are designed to offer consistent and well-defined methods compatible with relational databases. Developers can rely on these repositories to perform common data access tasks.
+
+==== Query Methods
 
-2. **Built-In Repositories:** Jakarta Data's built-in repositories, such as `PageableRepository` and `BasicRepository`, are designed to offer consistent and well-defined methods compatible with relational databases. Developers can rely on these repositories to perform common data access tasks.
+Jakarta Data's support for query methods, including pagination, ordering, and limiting, is designed to work seamlessly with relational databases.
 
-3. **Query Methods:** Jakarta Data's support for query methods, including pagination, ordering, and limiting, is designed to work seamlessly with relational databases.
+All limitations of Jakarta Persistence, the respective Jakarta Persistence provider, JPQL, SQL, and of the database apply when using entities that are annotated with `jakarta.persistence.Entity`. Repository methods must correspond to operations that are valid via `JPQL` and `SQL` queries. For example, although one can write a repository method that asks for sorting by a collection attribute or attempts to perform a Like operation on a numeric type rather than a String, there is no expectation for a Jakarta Data provider to support doing so because Jakarta Persistence and JPQL and SQL do not offer the ability.
```

Comment:
```suggestion
All limitations of Jakarta Persistence, the respective Jakarta Persistence provider, JPQL, SQL, and of the database apply when using entities that are annotated with `jakarta.persistence.Entity`. Repository methods must correspond to operations that are valid via `JPQL` and `SQL` queries. For example, although one can write a repository method that asks for sorting by a collection attribute or attempts to perform a Like operation on a numeric type rather than a String, there is no expectation for a Jakarta Data provider to support doing so because Jakarta Persistence, JPQL, and SQL do not offer the ability.
```